src/main/java/org/veupathdb/service/eda/subset/model/Entity.java

```diff
@@ -147,7 +148,15 @@ public Variable getVariableOrThrow(String variableId) {
   public List<String> getAncestorFullPkColNames() {
     return Collections.unmodifiableList(ancestorFullPkColNames);
   }
-  
+
+  public void setAttributesTableExists(boolean attributesTableExists) {
+    this.attributesTableExists = attributesTableExists;
+  }
+
+  public boolean isAttributesTableExists() {
+    return attributesTableExists;
```

Comment:
I forget what the retention is of these Entity objects.  I guess we are clearing entire studies when we reload using the admin endpoint?  If so, this is probably fine- just need to make sure we remember to reset if this table newly appears (don't know under what circumstances that happens either).

Reply:
They live in the cache, but it should get cleared automatically if the study gets reloaded